packages/patterns/chatbot-note.tsx

```diff
@@ -171,13 +171,12 @@ const ChatbotNote = recipe<LLMTestInput, LLMTestResult>(
 
     // why does MentionableCharm behave differently than any here?
     // perhaps optional properties?
-    const computeBacklinks = lift(
-      toSchema<
-        { allCharms: Cell<any[]>; content: Cell<string> }
-      >(),
-      toSchema<any[]>(),
+    const computeBacklinks = lift<
```

Comment:
meta style comment, and either should way works, but i wonder whether writing this as 

`lift(({ allCharms, content }: { allCharms: Cell<MentionableCharm[]>; content: Cell<string> }): any[] => { ... }`

is cleaner, and let's us sometimes leverage typescript inference (e.g. the return type here)